src/core.ts

```diff
@@ -20,9 +21,10 @@ let startTime: number;
 let activePlugins: IPlugin[];
 let bindings: IBindingContainer;
 let droppedPayloads: { [key: number]: IDroppedPayloadInfo };
+let pendingEvents: IEvent[] = [];
```

Comment:
>pendingEvents [](start = 4, length = 13)

Can you add a comment about what this is doing?

Reply:
Done

---
In reply to: [137239828](https://github.com/Microsoft/clarity-js/pull/28#discussion_r137239828) [](ancestors = 137239828)